pages/docs/webdev.md

```diff
@@ -11,7 +11,9 @@ The website source is available at <https://github.com/iris-hep/iris-hep.github.
 
 You can always click the edit button to make small edits to the website source, but if you want to test locally or make larger edits, you'll want to clone the source for the website and build it with Ruby.
 
-### Installing Ruby
+### Manual environment control
+
+#### Installing Ruby
```

Comment:
@henryiii if you don't have any concerns/dislikes about the way that I've restructured the docs here I might go ahead and merge this as is and then we can revise later if people find things confusing.